macos/ds_store.ksy

```diff
@@ -0,0 +1,193 @@
+meta:
+  id: ds_store
+  title: macOS '.DS_Store' format
+  license: CC-BY-SA-4.0
```

Comment:
CC-BY-* are not very suitable licenses for source code. They are more for art.

Reply:
Right. I've found a lot of CC-licensed KSY files in the file format gallery. I thought it was a kind of convention to use such licenses. I changed to MIT.

Reply:
CC-0 is different from other CC licences, it is just a public domain disclaimer with some legal words making it suitable for the jurisdictions like Germany.

Reply:
Interesting, I didn't know. Thanks for the explanation.